lib/crypto/crypto_scrypt.c

```diff
@@ -28,7 +28,6 @@
  */
 #include "scrypt_platform.h"
 
-#include <sys/types.h>
```

Comment:
This was intentional -- older UNIX systems require `<sys/types.h>` for mmap, even though the latest versions of POSIX say that `<sys/mman.h>` should be the only required header.